diff --git a/node_modules/three/src/core/Object3D.d.ts b/node_modules/three/src/core/Object3D.d.ts
index 76608a8..4a74d13 100644
--- a/node_modules/three/src/core/Object3D.d.ts
+++ b/node_modules/three/src/core/Object3D.d.ts
@@ -14,6 +14,7 @@ import { Raycaster } from './Raycaster';
 import { EventDispatcher } from './EventDispatcher';
 import { BufferGeometry } from './BufferGeometry';
 import { Intersection } from './Raycaster';
+import { Skeleton } from "../objects/Skeleton";
 
 export let Object3DIdCount: number;
 
@@ -180,6 +181,8 @@ export class Object3D extends EventDispatcher {
 	 */
 	readonly isObject3D: true;
 
+  geometry: Geometry | BufferGeometry;
+
 	/**
 	 * Calls before rendering object
 	 */
